cassandra/murmur3.py

```diff
@@ -0,0 +1,110 @@
+from six.moves import range
+import struct
+
+
+def body_and_tail(data):
+    l = len(data)
+    nblocks = l // 16
+    tail = l % 16
+    if nblocks:
+        return struct.unpack_from('qq' * nblocks, data), struct.unpack_from('b' * tail, data, -tail), l
+    else:
+        return tuple(), struct.unpack_from('b' * tail, data, -tail), l
+
+
+def rotl64(x, r):
```

Comment:
Probably not important because nothing "rotl"s anything more than 33. Anything that >= 64 is incorrect.
>>> hex(rotl64(1, 64))
'0x10000000000000001L'